dataland-frontend/src/components/general/apiKey/ApiKeyCard.vue

```diff
@@ -16,12 +16,8 @@
       <div class="flex justify-content-between mb-3">
         <div>
           <div class="text-900 font-medium text-xl text-left">API Key info</div>
-          <span :class="{ 'text-red-700': !isKeyExpired }" class="block text-600 mb-3 mt-6">
-            {{
-              isKeyExpired
-                ? `The API Key will expire on ${formatExpiryDate(expiryDateInDays)}`
-                : `The API Key expired ${formatExpiryDate(expiryDateInDays)}`
-            }}
+          <span :class="{ 'text-red-700': !isKeyExpired() }" class="block text-600 mb-3 mt-6">
+            {{ whenKeyExpire }}
           </span>
         </div>
 
```

Comment:
Just a naming side note:

I noticed that boths expressions:  "expire date" and "expiry date" are used thorughout the code.

I'd suggest to make it consistent and use "expiry date" everywhere.

Reply:
ok

Reply:
Looks good for ApiKeysPage.vue and ApiKeyCard.vue.

You could do this refactoring for CreateApiKeyCard.vue too, then it would be consistent everywhere.
If you do so, remeber to also refactor this in the tests to be consistent and avoid selector-errors.

Reply:
We are sure we want change "The API Key will expire on ... " to " The API Key will expiry on ... " ?

Reply:
No, in that sentence "expire" is correct.

But we still have things like "expireTimeDays", "expireTimeDropdown" , a label "expireTime", a data-test html tag value "expireDataPicker" and an id "expireTimeWrapper" in CreateApiKeyCard.vue that could be refactored.

Reply:
"expiryDatePicker" might be better than "expiryDataPicker"